pkg/signatures/celsig/config.go

```diff
@@ -105,32 +103,26 @@ func NewConfigsFromDir(dirPath string) ([]SignaturesConfig, error) {
 func walkFilesWithExtensions(rootDir string, extensions []string) ([]string, error) {
 	var files []string
 
-	err := capabilities.GetInstance().Specific(
```

Comment:
Why do we remove this request?

Reply:
Ok, I see why from you commit comment.
I'm just trying to think what should be done in the future if we need to call this function in runtime (maybe we will support dynamically loading new signatures?)
Can't we move the capabilities initialization part to be before calling this func?

Reply:
> I'm just trying to think what should be done in the future if we need to call this function in runtime (maybe we will support dynamically loading new signatures?)

Yes, that will be the "next step", together with the "mapping all things that need to be done for the dynamic unload/load of signatures" (for the API). For now I thought it would be ok to keep it like this and fix the bugs we currently have (For the release). WDYT ?